### ISSUE TITLE
test datas

### ISSUE THREAD

**Issue by cystal-lili** [HuguesTHOMAS/KPConv#132]
Hello, I am running the S3DIS data set. The training code has run through. I used the second block as the test set. Do I need to modify anything in the test code?

**Comment by cystal-lili** [comment]
hello, self.cloud_names = ['Area_1', 'Area_2', 'Area_3', 'Area_4', 'Area_5', 'Area_6']

        self.all_splits = [0, 1, 2, 3, 4, 5]

        self.validation_split = 1
I have modified this piece, but should the code in utils/tester be modified?
eg:
                        if dataset.name.startswith('Semantic3D'):

                            ascii_name = join(test_path, 'predictions', dataset.ascii_files[cloud_name])

                        else:

                            ascii_name = join(test_path, 'predictions', cloud_name[:-4] + '.txt')

                        np.savetxt(ascii_name, preds, fmt='%d')

                        i_test += 1

**Comment by HuguesTHOMAS** [comment]
Hi @cystal-lili,

Your modification is good and you should not modify anything in `tester.py`. But make sure you used this modification also during training, otherwise, your results will not be valid.

Best,
Hugues
